adapters/couchdb.md

```diff
@@ -0,0 +1,82 @@
+# CouchDB CloudEvents Adapter
+
+This document describes how to convert:
+- [CouchDB Document events](http://docs.couchdb.org/en/stable/api/database/changes.html) and
+- [CouchDB Database events](http://docs.couchdb.org/en/stable/api/server/common.html#db-updates)
+
+into CloudEvents.
+
+Each section below describes how to determine the CloudEvents attributes
+based on the event type.
+
+## /db/_changes
```

Comment:
I think the `_` makes everything else after this show up in _italics_.  I think if you escape it (e.g. `\_`) then it will fix that.

Reply:
fixed